meta-xt-prod-cockpit-rcar-driver-domain/recipes-extended/vis-service/files/VisClient.cpp

```diff
@@ -84,6 +84,33 @@ VisClient::VisClient(QObject *parent, const QString &url, const QString& rpmsg):
     data.value = 0;
     write(mFdept, &data, sizeof(data));
     qDebug() << "Create VIS client - send 0 to reset";
+
+    qDebug() << "!!!! RESET VALUES:" << mUrl;
+    data.value = 0;
+    data.ioctl_cmd = CtlIO_id::DOOR_OPEN;
```

Comment:
Maybe I'm wrong, but CtlIO_id is defined in lines 28-33 as enum with only 4 values: SPEED, GEAR, RPM, TURN.
And here we are using 12 not defined values.
Are you sure, that this code works as expected?